test/benchmarks/bench_classify.rb

```diff
@@ -39,31 +39,25 @@ def setup
 
   def bench_allocations_with_cache_disabled
     Primer::Classify::Cache.instance.clear!
+    Primer::Classify::AttrCache.instance.clear!
 
-    assert_allocations "3.0" => 46, "2.7" => 46, "2.6" => 49, "2.5" => 50 do
+    assert_allocations "3.0" => 47, "2.7" => 46, "2.6" => 48, "2.5" => 49 do
       Primer::Classify::Cache.instance.disable do
-        Primer::Classify.call(**@values)
+        Primer::Classify::AttrCache.instance.disable do
+          Primer::Classify.call(**@values)
+        end
       end
     end
-  ensure
-    Primer::Classify::Cache.instance.preload!
   end
 
-  def bench_allocations_with_cache_preloaded
-    Primer::Classify::Cache.instance.clear!
-    Primer::Classify::Cache.instance.preload!
-
-    assert_allocations "3.0" => 42, "2.7" => 35, "2.6" => 38, "2.5" => 39 do
-      Primer::Classify.call(**@values)
-    end
-  end
+  def bench_allocations_with_cache_warmed
```

Comment:
Since we know have two forms of caching (preloading + an LRU cache) should there be two separate tests?

Reply:
Hmm that's a good call, I can put in another one.